src/stories/DwollaBalanceDisplay.stories.tsx

```diff
@@ -28,7 +28,7 @@ export default {
               {
                 _embedded: {
                   balance: {
-                    balance: { value: '100.00', currency: 'USD' }
```

Comment:
Updating a story to trigger storybook deploy automatically on merge, and as another layer of assurance that the deployment was successful.